bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -300,26 +300,24 @@ limitations under the License.
       <!-- Required properties:
          google.bigtable.project.id
          google.bigtable.instance.id
-         google.dataflow.stagingLocation
-         cloud.test.data.folder
+         google.dataflow.gcsPath
      -->
       <build>
         <plugins>
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-failsafe-plugin</artifactId>
+            <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
why surefire? shouldnt we use failsafe for integration tests?

Reply:
`surefire` to be similar to other integration tests:
- https://github.com/googleapis/java-bigtable-hbase/blob/master/bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/pom.xml#L42-L51
- https://github.com/googleapis/java-bigtable-hbase/blob/master/bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/pom.xml#L50-L56
- https://github.com/googleapis/java-bigtable-hbase/blob/master/bigtable-dataflow-parent/bigtable-beam-import/pom.xml#L336-L342

there must be some reasons why `surefire` is used.